packages/renderer/src/lib/onboarding/OnboardingItem.svelte

```diff
@@ -39,22 +43,36 @@ onMount(() => {
     }
   });
 
-  const itemHtml = replacePlaceholders(item.value);
-  html = itemHtml;
+  contextsUnsubscribe = context.subscribe(value => {
+    globalContext = value;
+    updateHtml();
+  });
+
+  updateHtml();
 });
 
+onDestroy(() => {
+  contextsUnsubscribe?.();
+});
+
+function updateHtml() {
```

Comment:
```suggestion
function updateHtml(): void {
```

it is missing return type